types.d.ts

```diff
@@ -0,0 +1,78 @@
+interface IPerseverantOptions<TSerializer = typeof JSON> {
+  /**
+   * Name of the storage. Default is `'global'`
+   * Hightly suggested to use non-default name
+   */
+  name?: string;
+  /**
+   * Name of storage folder. Default is `'HOME/.perseverant'`
```

Comment:
forgot this one ... can you please make it `$HOME/.perseverant` to better indicate is an env like path ? thanks!

Reply:
I think I've done that already